### ISSUE TITLE
RSC_007 doesn't say which file exactly is missing

### ISSUE THREAD

**Issue by tofi86** [IDPF/epubcheck#475]
Message `RSC_007` doesn't say which resource exactly can't be found in the EPUB:

```
ERROR(RSC-007): /Users/tobiasfischer/Entwicklung/EPUB-Kram/_testdaten/official/UnmanifestedGuideItems.epub/OEBPS/content.opf(29,67):
  Referenced resource is not found in the ePub.
```

A better message would be:

```
ERROR(RSC-007): /Users/tobiasfischer/Entwicklung/EPUB-Kram/_testdaten/official/UnmanifestedGuideItems.epub/OEBPS/content.opf(29,67):
  Referenced resource "toc.html" is not found in the ePub.
```

**Comment by tofi86** [comment]
references #588, #634

**Comment by tofi86** [comment]
Added a fix for that in PR #722
